src/content.ts

```diff
@@ -441,7 +441,7 @@ async function videoIDChange(id): Promise<void> {
     //close popup
     closeInfoMenu();
```

Comment:
If we are calling update on the popup, can this be removed?

Reply:
Did some testing, seems like this can indeed be safely removed now.
The popup correctly persists and updates between video changes (manual, playlist, autoplay and even going to the home/search page)